Woof/Woof/ViewModel/OwnerCardViewModel.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/// The view model for a pet owner card, responsible for preparing and providing data for the view.
+final class OwnerCardViewModel: ObservableObject {
+    /// The full name of the pet owner.
+    @Published var fullName: String
+
+    /// The phone number of the pet owner.
+    @Published var phoneNumber: String
+
+    /// The address of the pet owner.
+    @Published var address: String
+
+    /// The URL of the pet owner's avatar image.
+    @Published var imageURL: URL?
+
+    /// Initializes a new instance of the pet owner card view model with the provided owner.
+    ///
+    /// - Parameter owner: The owner for which the card view model is created.
+    init(owner: Owner) {
+        self.owner = owner
```

Comment:
you expose the owner, that brokes MVVM pattern.
```suggestion
    /// - Parameter owner: The owner for which the card view model is created.
    init() {
        self.owner = Owner.Dummy.laraCroft
```

Reply:
The file are deleted. Thanks for the comment!